### ISSUE TITLE
`--touchpad-support 0` in genzehn fails on CX

### ISSUE THREAD

**Issue by Streetwalrus** [ndless-nspire/Ndless#42]
According to `--help`, `--touchpad-support arg (=1)  Whether touchpads (classic) are supported`, however if I turn support off and try to run the program on a CX, ndless refuses to saying touchpads are not supported.

**Comment by Vogtinator** [comment]
Yeah, the explanation is wrong, it should be "(classic and CX)". It tests for ```is_touchpad``` in https://github.com/ndless-nspire/Ndless/blob/master/ndless/src/resources/zehn_loader.cpp#L221

If you want to indicate that non-CX is not supported, use ```--color-support``` and/or ```--32MB-support```.

**Comment by Streetwalrus** [comment]
As far as I can tell there is no way to filter out classics without blocking CMs, though. I don't support classics due to the many hardware differences and lack of testing hardware.

**Comment by Vogtinator** [comment]
```--color-support``` is for exactly that purpose.

**Comment by Streetwalrus** [comment]
That's not what the code and description say, setting it to false would make it refuse to run on color models.

**Comment by Vogtinator** [comment]
Yeah, I realized too late that I had a knot in my brain...

There needs to be a new flag for that, "--classic-support". In the meantime you can set ```--clickpad-support``` to false and do a manual check for touchpads...

Can you do a PR for the flag? I'm currently very busy.

**Comment by Streetwalrus** [comment]
No problem, I'll take care of it.

**Comment by Vogtinator** [comment]
BTW: You should do a manual check anyway. Zehn flags are no guarantee that the application won't start on certain models, the minimal PRG wrapper does not implement model checking (yet, it can certainly be implemented).

**Comment by Streetwalrus** [comment]
It's enough for my own purposes, I'm about to use APIs from r2005 for the new LCD layout so no point in distributing PRG binaries.
